src/api/spec/helpers/webui/webui_helper_spec.rb

```diff
@@ -331,7 +385,60 @@
   end
 
   describe '#setup_codemirror_editor' do
-    skip('Please add some tests')
+    it 'sets the style for the editor' do
```

Comment:
Actually I don't think that mocking everything away makes much sense for testing here. I still think we should move the content_for to the view and only test that the uuid gets increased.